react-native-plaid-link-sdk.podspec

```diff
@@ -16,5 +16,5 @@ Pod::Spec.new do |s|
   s.source_files  = "ios/*.{h,m}"
 
   s.dependency 'React-Core'
-  s.dependency 'Plaid', '~> 2.5.1'
+  s.dependency 'Plaid', '~> 3.1.0'
```

Comment:
this is a major version bump which means we'd need to also do a major RN version bump. Is it required?

Reply:
Good catch, I'll make this a major bump